d/doctrine_cache/doctrine-cache_1.10.0_1.10.2_rhel_8.4.sh

```diff
@@ -0,0 +1,52 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: cache
+# Version	: 1.10.0, 1.10.2
```

Comment:
As per line 18, build script is created for version 1.10.0. Please update accordingly

Reply:
The script is tested and working fine on rhel 8.4. Also parameterized the version in line 18.